tutorials/outflux.md

```diff
@@ -0,0 +1,174 @@
+# Gettin started with Outflux
+
+Designed to help users seamlessly migrate from InfluxDB to TimescaleDB, Outflux enables users to pipe exported data directly into TimescaleDB. Outflux manages the schema discovery, validation, and creation for you. It’s easy to use, configurable, and most importantly, it’s fast.
+
+## Instalation
+
+### Before we start
+
+Before we start, you will need the following setup:
+* A running instance of InfluxDB at a known location and a means to connect to it
+* TimescaleDB installed and a means to connect to it (visit our [Docs page to learn how to set it up](https://docs.timescale.com/v1.2/getting-started/installation))
```

Comment:
The format for docs is to use footer links `[link text][link-footer-ref]`.  That way it's easier to change, and identify changes to link addresses.

Reply:
I won't point out all of the places that it needs to be switched, but all links should be in this format.  The only exception are anchor links on the same page.